playbook/app/pb_kits/playbook/pb_map/_map_controls.tsx

```diff
@@ -12,6 +13,8 @@ type MapControlTypes = {
   children?: React.ReactNode | React.ReactNode[]
 }
 
+const eyeIcon = getAllIcons()["eye"]
```

Comment:
Hey @markdoeswork looks like your icon is changing the icon look and color from what's on prod, was this intentional? We might need designer signoff here:

![Screenshot 2024-09-17 at 2 30 11 PM](https://github.com/user-attachments/assets/4f118941-39c4-4bf7-bcc9-1fcca2b2d946)

Left is prod, right is your review env